config/alphabet.conf

```
@@ -27,186 +27,186 @@ declare -a alphaC=(
 	" ╚═════╝",
 )
 declare -a alphaD=(
-	"",
-	"",
-	"",
-	"",
-	"",
-	"",
+	"██████╗ ",
+	"██╔══██╗",
+	"██║  ██║",
+	"██║  ██║",
+	"██████╔╝",
+	"╚═════╝ ",
 )
 declare -a alphaE=(
-	"",
-	"",
-	"",
-	"",
-	"",
-	"",
+	"███████╗",
+	"██╔════╝",
+	"█████╗  ",
+	"██╔══╝  ",
+	"███████╗",
+	"╚══════╝",
 )
 declare -a alphaF=(
-	"",
-	"",
-	"",
-	"",
-	"",
-	"",
+	"███████╗",
+	"██╔════╝",
+	"█████╗  ",
+	"██╔══╝  ",
+	"██║     ",
+	"╚═╝     ",
 )
 declare -a alphaG=(
-	"",
-	"",
-	"",
-	"",
-	"",
-	"",
+	" ██████╗ ",
+	"██╔════╝ ",
+	"██║  ███╗",
+	"██║   ██║",
+	"╚██████╔╝",
+	" ╚═════╝",
```

* All lines should be same length
```suggestion
	" ╚═════╝ ",
```